src/migrations/versions/c5d3236b81a3_db_script.py

```diff
@@ -0,0 +1,31 @@
+"""db_script
+
+Revision ID: c5d3236b81a3
+Revises: 
+Create Date: 2022-07-27 09:05:34.676045
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'c5d3236b81a3'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+engine = sa.create_engine('postgresql://postgres:1234@localhost/ea_restaurant')
```

Comment:
Why do you need this here? If you already add your db url above?

Reply:
done.